kube_dns/README.md

```diff
@@ -68,10 +60,4 @@ See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/kube
 The Kube-DNS check does not include any event at this time.
 
 ### Service Checks
-The Kube-DNS check does not include any service check at this time.
-
-## Troubleshooting
```

Comment:
You probably did not want to remove the troubleshooting and further reading sections in this README ?